tools/ratbagc.py.in

```diff
@@ -262,6 +262,12 @@ class RatbagdDevice(metaclass=MetaRatbag):
         """The device name, usually provided by the kernel."""
         return libratbag.ratbag_device_get_name(self._device)
 
+    @property
+    def device_type(self):
+        """The type of this device: unspecified(0), mouse(1), keyboard(2) or other(3)"""
+        return libratbag.ratbag_device_get_device_type(self._device)
+
+
```

Comment:
The enum should be defined in `ratbagc.py` as well, the behavior of this function should be the same in both `ratbagd.py` and `ratbagc.py`.

Reply:
will fix, did not know that.